penguin/web/services/opencode_events.py

```
@@ -0,0 +1,224 @@
+            await emit_opencode_event(core_getter(), event_type, properties)
+        except Exception:
+            if logger:
+                logger.debug(
+                    "Failed to emit opencode event %s",
+                    event_type,
+                    exc_info=True,
+                )
+
+    try:
+        loop = asyncio.get_running_loop()
+        loop.create_task(_runner())
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)